docs/json_for_test/invalid-[.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Actually, there is no need to check syntactically invalid jsons (json parsing library already does that well). I suggest to check semantically invalid json (json that contains unexpected values and does not describe layer structure)

Reply:
Yes, I understand, I'm working on it